### PR TITLE
Readme build little clarification

### LINKED ISSUE #3818: Easy way to build the game for tests purpose
Hi,

I am willing to contribute (at my little level) so I'm trying, at first, to build the game pulled from the master branch (should go fine).

But, in my opinion, this very little task is already not that simple or maybe it could be usefull to add few little more information in the readme.

In my case, all the automatic tests are running fine under visual studio but when I try to "generate osu.game" I can't find any executable file generated to launch the game.
On the other side, if I try to run "dotnet run --project osu.Desktop" it stays blocked on "Loading shader Texture2D/Texture" for hours. 

So I guess I am doing something wrong or something could be more clear in the instruction from the readme. 

I guess the readme is already clear for every one who is used to the project or to visual Studio but not really for every beginner.

### PR DESCRIPTION
**Pull request by darkyne**
Very little proposition I suggest to make it clearer :)

In my case, I searched a long time for an exe file for no reason :D

Resolves #3818

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Clone the repository including submodules
 
 Build and run
 
-- Using Visual Studio 2017, Rider or Visual Studio Code (configurations are included)
+- Using Visual Studio 2017, Rider or Visual Studio Code (configurations are included. No exe file will be generated, simply "dotnet restore" in osu folder then "dotnet run" in osu.desktop folder)
```
Comment (jorolf) [inline README.md]: Move this a line down. All of those IDEs do this automatically. ``dotnet restore`` is also automatically executed by ``dotnet run``

Comment (darkyne) [reply README.md]: Not 100% sure what you meant by moving a line down.
Sorry, I hope this is fine now.

**Comment by peppy** [comment]
I do not think this adds any clarity.

**Comment by peppy** [comment]
I mean what was the actual issue? Because the next line in the readme tells you to use `dotnet run` which would run the project.

**Comment by darkyne** [comment]
When I read the readme, I feel like: There is different options: Running directly via visual studio or (other option) via terminal. I didn't understand it was actually steps :D
But I agree I went particuarly dump this time...

**Comment by peppy** [comment]
Closing this for now, then.

**Comment by UselessToucan** [comment]
You can get exe file by specifying the runtime. Command example "dotnet build -c Release -r win-x64".

**Pull request closed without merging**